src/System.Data.SqlClient/src/System/Data/SqlClient/SqlConnection.cs

```diff
@@ -1522,16 +1522,24 @@ private static void ChangePassword(string connectionString, SqlConnectionString
         // this only happens once per connection
         // SxS: using named file mapping APIs
 
-        internal void RegisterForConnectionCloseNotification<T>(ref Task<T> outerTask, object value, int tag)
+        internal void RegisterForConnectionCloseNotification<T>(ref Task<T> outerTask, object value, int tag,Func<Task<T>,object,Task<T>> callback)
```

Comment:
```suggestion
        internal void RegisterForConnectionCloseNotification<T>(ref Task<T> outerTask, object value, int tag, Func<Task<T>, object,Task<T>> callback)
```

Reply:
Done. 

At some point it might be worth ,e just going through the entire solution and applying the repo formatting guidelines. If I format entire files at the moment I quite often get formatting fixes in bits of code I haven't changed which I know people don't like when reviewing.